funfact/initializers.py

```diff
@@ -30,6 +30,23 @@ def __call__(self, shape):
         return ab.ones(shape, self.dtype)
 
 
+class Eye:
+    '''Initializes diagonal elements to 1 and all others to 0.
+
+    Args:
+        dtype: Numerical type of elements.
+    '''
+    def __init__(self, dtype=None):
+        self.dtype = dtype or ab.float32
+
+    def __call__(self, shape):
+        if len(shape) != 2 or shape[0] != shape[1]:
```

Comment:
the `eye` methods in torch and numpy both support rectangular identity matrices with a second input argument (m and M respectively), We could also support that by forwarding this to the backends?

Reply:
Sure!